zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -299,7 +299,7 @@ <h3 class="interpreter-title">{{setting.name}}
             <td>
               <textarea ng-if="valueform.$visible" ng-model="dep.exclusions"
                         placeholder="(Optional) comma separated groupId:artifactId list"
-                        form="valueform"
+                        e-form="valueform"
```

Comment:
@corneadoug Ah didn't know that. I'm using IntelliJ now, it said like below for `form="valueform"`.
<img width="507" alt="screen shot 2016-09-19 at 4 38 50 pm" src="https://cloud.githubusercontent.com/assets/10060731/18625174/a9aac3ea-7e87-11e6-8de9-02d585dab979.png">

Anyway I reverted it :)